management/templates/system-backup.html

```diff
@@ -111,6 +112,31 @@ <h3>Configuration</h3>
       <input type="text" class="form-control" rows="1" id="backup-target-pass">
     </div>
   </div>
+  <!-- Backblaze -->
+  <div class="form-group backup-target-b2">
+      <div class="col-sm-10 col-sm-offset-2">
+      <p>Backups are stored in a Backblaze B2 bucket. You must have a Backblaze account already.</p>
+      <p>You MUST manually copy the encryption password from <tt class="backup-encpassword-file"></tt> to a safe and secure location. You will need this file to decrypt backup files. It is NOT stored in your Backblaze B2 bucket.</p>
+      </div>
+  </div>
+  <div class="form-group backup-target-b2">
+      <label for="backup-target-b2-user" class="col-sm-2 control-label">B2 Application KeyID</label>
+      <div class="col-sm-8">
+      <input type="text" class="form-control" rows="1" id="backup-target-b2-user">
```

Comment:
Some nested elements (input, p) are not properly indented in this patch. It would be nice if you could add indents for these

Reply:
Good catch :)